independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/BlockingDefault.java

```diff
@@ -0,0 +1,10 @@
+package org.jboss.resteasy.reactive.common.processor;
+
+public enum BlockingDefault {
```

Comment:
Do we really want this to be able to change this?

Reply:
Not sure I understand your question :(

Reply:
IIUC, this allows a user to set what the default handling should be, no?